queue_job/views/queue_job_views.xml

```diff
@@ -15,19 +15,25 @@
                         type="object"
                         groups="queue_job.group_queue_job_manager"/>
                     <button name="button_done"
-                        states="pending,enqueued,failed"
+                        states="pending,enqueued"
```

Comment:
Depending on the meaning of 'Done' (I'd say, has been executed manually outside of the queue and perhaps even outside of Odoo), you might still want to set 'failed' and even 'cancelled' jobs to done. Or do you have another interpretation?

Reply:
I followed point 3 from https://github.com/OCA/queue/issues/342#issuecomment-833548686, I personally would show the button everywhere

Reply:
Ah, I made that suggestion with the idea to replace the 'Set Done' button entirely with a 'Set Cancelled' button, which is probably not a good idea in a stable version or not at all depending on the semantics so please show the button everywhere.